### DISCUSSION TITLE
2006/09/30/%E7%94%A8%E6%88%B7%E8%87%AA%E5%AE%9A%E4%B9%89%E6%8E%A7%E4%BB%B6%E4%B8%AD-control-type-def-%E5%92%8C-strict-type-def-%E7%9A%84%E5%8C%BA%E5%88%AB

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#118] [Announcements]
# 2006/09/30/%E7%94%A8%E6%88%B7%E8%87%AA%E5%AE%9A%E4%B9%89%E6%8E%A7%E4%BB%B6%E4%B8%AD-control-type-def-%E5%92%8C-strict-type-def-%E7%9A%84%E5%8C%BA%E5%88%AB

https://lv.qizhen.xyz/

https://blog.qizhen.xyz/2006/09/30/%E7%94%A8%E6%88%B7%E8%87%AA%E5%AE%9A%E4%B9%89%E6%8E%A7%E4%BB%B6%E4%B8%AD-control-type-def-%E5%92%8C-strict-type-def-%E7%9A%84%E5%8C%BA%E5%88%AB

**Comment by ruanqizhen** [comment]
之前博客下的留言：


陈小刚
2011年03月30日 16:26 编辑
用严格类型定义的时候，似乎有一些控件其实例的常量也是与定义控件相关的，比如枚举型控件

回复
ruanqizhen
2011年03月31日 09:23 编辑
确实是这样的，现在的LabVIEW在这上面有点乱。Enumerate常量会变，而Ring常量不会变。

回复
